telemetry/definitions/commonDefinitions.json

```
@@ -1058,6 +1058,11 @@
             "type": "boolean",
             "description": "Whether this was an individual point or an aggregation of points."
         },
+        {
+            "name": "isReAuth",
+            "type": "boolean",
+            "description": "If reauthentication was done"
```

What does this mean?
```suggestion
            "description": "If an existing connection was reauthenticated (as opposed to ...)"
```

---

Couldn't really think of a good description. This is basically a flag for if reauthentication was done during the createToken process. I've reworded, lmk if it improves.

---

Main goal is to help other Toolkits apply this, so any description that helps them understand when this is relevant, is helpful. 

Mentioning "during createToken" is probably worth adding